Sources/SourceKitLSP/Swift/SemanticTokens.swift

```diff
@@ -58,7 +58,7 @@ extension SwiftLanguageService {
       if let range = range.flatMap({ snapshot.byteSourceRange(of: $0) }) {
         range
       } else {
-        ByteSourceRange(offset: 0, length: await tree.totalLength.utf8Length)
+        await tree.range
```

Comment:
@ahoppen we await here but also on line 65. 
Would it not be best to change `async let tree = syntaxTreeManager.syntaxTree(for: snapshot)` -> `let tree = await syntaxTreeManager.syntaxTree(for: snapshot)`

Reply:
The benefit of `async let` is allow us to compute the syntax tree and semantic highlighting tokens in parallel. If we await `syntaxTreeManager.syntaxTree(for:)`, we would wait for the syntax tree to be computed before starting the computation of semantic tokens.